crates/youki/src/main.rs

```diff
@@ -38,13 +38,15 @@ use nix::unistd::getuid;
 #[derive(Parser, Debug)]
 #[clap(version = crate_version!(), author = "youki team")]
 struct Opts {
-    /// root directory to store container state
-    #[clap(short, long)]
-    root: Option<PathBuf>,
+    #[clap(long)]
+    debug: bool,
```

Comment:
should this be an option as well? The `--debug` flag can be true, false, or absent.

Reply:
I noticed that there is no default value for --systemd-cgroup either, should I add one?

Reply:
> should this be an option as well? The `--debug` flag can be true, false, or absent. Does the variable default to `false`?

Variable is false when flag is not used

Reply:
If the variable defaults to false, then we are good.